custom_configs/download_video_configuration.yml

```diff
@@ -0,0 +1,3 @@
+- - executor: .database.FirebaseSource.FirebaseSource
```

Comment:
this config will fail because the format is different now:
https://github.com/AutoDash/AutoDash/blob/master/default_configuration.yml

Reply:
True, I'll copy the new format